sound/soc/sof/ipc4-topology.h

```diff
@@ -166,18 +166,39 @@ struct sof_ipc4_pin_format {
 	struct sof_ipc4_audio_format audio_fmt;
 };
 
+/**
+ * struct sof_ipc4_tplg_mod_cfg - module configuration entry from topology
+ * @ibs: Input Buffer Size
+ * @obs: Output Buffer Size
+ * @cpc: Cycles Per Chunk
```

Comment:
what is a chunk

Reply:
a chunk that is processed at once, typically 1ms.

But we don't really care in kernel, we read the number from manifest/topology and pass it to the firmware and we trust that they are added correctly.
We don't verify the ibs/obs numbers either for that matter.